docs/controller.md

```diff
@@ -1,40 +1,358 @@
 ## Controller
 
-In order to customize an action, we need to inherit the controller from `Wallaby::ResourcesController`. Then we can override the general resourceful actions (index/show/new/create/edit/update/destroy) or add an action:
+Wallaby allows devs to customize controller logics in a custom controller that inherits from `Wallaby::ResourcesController`.
+
+- [Declaration](#declaration)
+
+The following resourceful actions can be customized:
+
+- [Home](#home)
+- [Index](#index)
+- [Show](#show)
+- [New](#new)
+- [Create](#create)
+- [Edit](#edit)
+- [Update](#update)
+- [Destroy](#destroy)
+
+Also it is possible to customize the whitelisting parameters for mass assignment of create/update:
+
+- [`resource_params`](#resource_params)
+
+### Declaration
+
+Let's see how a controller can be created so that Wallaby knows its existence. Similar to what it is normally done in Rails, create a custom controller for model `Product` that inherits from `Wallaby::ResourcesController` as below:
 
 ```ruby
 #!app/controllers/products_controller.rb
 class ProductsController < Wallaby::ResourcesController
-  # def index; super; end
-  # def show; super; end
-  # def new; super; end
+end
+```
+
+> NOTE: although it inherits `Wallaby::ResourcesController`, it is possible to access to all methods in `ApplicationController`. Because `Wallaby::ResourcesController` inherits `ApplicationController` unless this is changed in Wallaby [authentication configuration](configuration.md#authentication).
+
+If `ProductsController` is taken, it is still possible to use other name but needs to create method `self.model_class` to specify the model that it relates to:
+
+```ruby
+#!app/controllers/admin/products_controller.rb
+class Admin::ProductsController < Wallaby::ResourcesController
+  def self.model_class
+    Product
+  end
+end
+```
+
+### Home
+
+`home` action is basically a blank action which renders the `home` template as the landing page of Wallaby:
```

Comment:
Will the engine mount path route to this action?

Reply:
yes, it does